Dockerfile

```diff
@@ -1,7 +1,19 @@
 FROM node:14-alpine
 
-RUN mkdir /config
+WORKDIR /monika
+
+COPY package*.json .
```

Comment:
This line gives me 
![image](https://user-images.githubusercontent.com/16670475/113688766-d209a980-96f3-11eb-8c96-096850bc1f63.png)

Reply:
I got same result.

Reply:
fixed